neo/core/__init__.py

```diff
@@ -35,6 +37,11 @@
 from neo.core.analogsignal import AnalogSignal
 from neo.core.irregularlysampledsignal import IrregularlySampledSignal
 
+# Import FilterClasses
+from neo.core.filters import Equal, IsIn, IsNot,\
```

Comment:
I don't think the filters should be imported into the top-level namespace,
rather just `from neo.core import filters`

Reply:
see: bf1c51da51def0edc9eaf2cbe6d8e1eb4e2931e3